### DISCUSSION TITLE
Mohon bantuan coding untuk shiny Apps checkboxGroupInput supaya bisa sum

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#242] [Q&A] [answered]
Halo,

Mohon bantuan codingnya untuk checkboxGroupInput supaya ketika dipilih lebih dari 1 bisa melakukan sum, pada contoh plot sbb menjadi error ketika dipilih lebih dari 1:
![image](https://user-images.githubusercontent.com/107389614/182405879-6d719796-ba29-460e-98aa-22f1d7d1c0ca.png)

Terima kasih

**Comment by rizkiajimahardika** [comment, marked as answer]
Baik pak, untuk dapat membuat check box untuk dapat memilih lebih dari 1, bapak bisa menggunakan fungsi checkboxGroupInput. 
`checkboxGroupInput(
  inputId,
  label,
  choices = NULL,
  selected = NULL,
  inline = FALSE,
  width = NULL,
  choiceNames = NULL,
  choiceValues = NULL
)`

Contohnya jika ingin menggunakan checkboxGroupInput :
`checkboxGroupInput("checkGroup1", label = h3("This is a Checkbox group"), 
                 choices = list("1" = 1, "2" = 2, "3" = 3),
                 selected = 1)`

Untuk lebih detailnya, bapak dapat melihat dokumentasi resminya di link berikut : https://shiny.rstudio.com/reference/shiny/1.6.0/checkboxGroupInput.html
